src/services/submissionService.ts

```diff
@@ -0,0 +1,1114 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
```

Comment:
I read through the healthcareProfessionalService file. Take a look at those comments as they will likely be the same in  submissionService and facilityService.